contracts/Arbitrum_SpokePool.sol

```diff
@@ -54,7 +55,7 @@ contract Arbitrum_SpokePool is SpokePool, CircleCCTPAdapter {
     }
 
     modifier onlyFromCrossDomainAdmin() {
-        require(msg.sender == _applyL1ToL2Alias(crossDomainAdmin), "ONLY_COUNTERPART_GATEWAY");
+        require(msg.sender == AddressUtils._applyL1ToL2Alias(crossDomainAdmin), "ONLY_COUNTERPART_GATEWAY");
```

Comment:
I think this library should be called CrossDomainAddressUtils as AddressUtils overlaps with the openzeppelin utility file that has utilities for any arbitrary EVM. This aliasing is very OVM/AVM specific

Reply:
https://github.com/across-protocol/contracts/pull/625/commits/24c1836a2a6c0db8ba04c13b8939cd738c4c9eb7